src/plugins/geomap/_base.scss

```diff
@@ -142,6 +142,7 @@
 
 .olControlAttribution {
 	bottom: 5px;
+	font-size: x-small;
```

Comment:
We do use xx-small as well in the Geomap CSS. Not ideal. We can change them all now or in the next release. Your call.